src/app/shared/components/template/components/task-card/task-card.component.scss

```diff
@@ -191,8 +191,10 @@
       border: 1px solid rgba(black, 0.07);
       filter: drop-shadow(var(--ion-default-box-shadow));
 
-      .img {
```

Comment:
woops 😅

Reply:
yeah, given that angular couples scss tightly with components would be nice if their vscode extension could also pick out unused classes. 

I can see one extension that claims to do that, but not tried myself:
https://marketplace.visualstudio.com/items?itemName=rcore.rcore-unused-css-classes